start.sh

```diff
@@ -12,4 +8,16 @@ then
   echo "fits never became ready. Exiting"
   exit 1
 fi
-bundle exec sidekiq -C config/sidekiq.yml -r ./listener.rb -vv
+
+echo "Updating Virus Definitaions"
+freshclam &
+
+if [ ${ENV:-production} == "development" ]; then
+  echo "Checking Bundle"
+  bundle check || bundle install
+  echo "Sleeping Forever"
+  sleep infinity
```

Comment:
Is the idea not to use Sidekiq in the development cycle?

Reply:
Yeah, and that's where the ENV variable comes in, so that a local developer gets the fits servlet but not sidekiq. if we wanted to run sidekiq we could shell in and run it like you would any other sidekiq process.

Reply:
I'm not seeing the need for this. Perhaps we could make this a different issue/PR? I doesn't seem germane to adding Clam.